### PR TITLE
docs: translation for `option` component

### LINKED ISSUE #424: `<option>`
Page: https://id.react.dev/reference/react-dom/components/option

Assignee: @aroyan

### PR DESCRIPTION
**Pull request by aroyan**
Related to  #330 

Closes #424

### CODE REVIEW

**Review by mazipan — APPROVED** [review]
Tinggal nunggu soal caveat aja nih yang masih rancu, overall okay dariku

File: src/content/reference/react-dom/components/option.md
Comment (resir014) [inline src/content/reference/react-dom/components/option.md:48]:
```suggestion
#### Caveats {/*caveats*/}
```

Untuk sementara, kita tetapkan kata *caveat*, sesuai [referensi diskusi ini](https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5751610)

cc: @mazipan @mhaidarhanif

File: src/content/reference/react-dom/components/option.md
Comment (resir014) [inline src/content/reference/react-dom/components/option.md:7]:
```suggestion
[Komponen `<option>` bawaan peramban](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) memungkinkan Anda untuk me-*render* opsi di dalam kotak [`<select>`](/reference/react-dom/components/select).
```

Italicise the word *render* as per the [Glossary](https://github.com/reactjs/id.reactjs.org/wiki/Glosarium).

File: src/content/reference/react-dom/components/option.md
Comment (resir014) [inline src/content/reference/react-dom/components/option.md:27]:
```suggestion
[Komponen `<option>` bawaan peramban](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) memungkinkan Anda untuk me-*render* opsi di dalam kotak [`<select>`](/reference/react-dom/components/select).
```

Idem.

File: src/content/reference/react-dom/components/option.md
Comment (resir014) [inline src/content/reference/react-dom/components/option.md:58]:
```suggestion
*Render* sebuah `<select>` dengan daftar komponen `<option>` di dalamnya untuk menampilkan sebuah kotak pilihan. Beri setiap `<option>` sebuah `value` yang merepresentasikan data yang akan dikirim dengan formulir.
```

**Review by resir014 — CHANGES_REQUESTED** [review]
Halo mas @aroyan, please take a look at these requested changes:

**Comment by aroyan** [comment]
> Halo mas @aroyan, please take a look at these requested changes:

Sure, it's updated now

File: src/content/reference/react-dom/components/option.md
Comment (mazipan) [inline src/content/reference/react-dom/components/option.md:50]: Always use `*` instead of `_` to italicized

File: src/content/reference/react-dom/components/option.md
Comment (mazipan) [inline src/content/reference/react-dom/components/option.md:50]:
```suggestion
* React tidak mendukung atribut `selected` dalam `<option>`. Sebagai gantinya, oper `value` dari opsi ke elemen induk [`<select defaultValue>`](/reference/react-dom/components/select#providing-an-initially-selected-option) untuk sebuah *uncontrolled select box*, atau [`<select value>`](/reference/react-dom/components/select#controlling-a-select-box-with-a-state-variable) untuk *controlled select*.
```

**Review by mazipan — APPROVED** [review]

**Comment by mazipan** [comment]
Asking peer review @r17x @mhaidarhanif @zainfathoni.
Also wait for @resir014 approval.

File: src/content/reference/react-dom/components/option.md
Comment (mhaidarhanif) [inline src/content/reference/react-dom/components/option.md:12]: I think the value (`someOption` and `otherOption`) shouldn't be translated, it has the same level as a variable name, let's keep it as it is. Only the UI text can be translated.

File: src/content/reference/react-dom/components/option.md
Comment (mhaidarhanif) [inline src/content/reference/react-dom/components/option.md:27]: The URL was changed, need to change back

```suggestion
[Komponen `<option>` bawaan peramban](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) memungkinkan Anda untuk me-*render* opsi di dalam kotak [`<select>`](/reference/react-dom/components/select).
```

**Review by mhaidarhanif — CHANGES_REQUESTED** [review]
Need to change.

File: src/content/reference/react-dom/components/option.md
Comment (aroyan) [reply src/content/reference/react-dom/components/option.md:27]: thanks

File: src/content/reference/react-dom/components/option.md
Comment (aroyan) [reply src/content/reference/react-dom/components/option.md:12]: what about value for this [FruitPicker component](https://github.com/reactjs/id.react.dev/pull/368/files#diff-f8e5a07bbbedc1f852d250d167b646f71fc5a5dcc8a76534c437db9c517df257R65)?

Comment (resir014) [reply src/content/reference/react-dom/components/option.md:12]: @aroyan I think we should roll back only the value to the original translation on that part.

File: src/content/reference/react-dom/components/option.md
Comment (resir014) [inline src/content/reference/react-dom/components/option.md:72]:
```suggestion
        <option value="apple">Apel</option>
        <option value="banana">Pisang</option>
        <option value="orange">Jeruk</option>
```

Please revert the option value.

File: src/content/reference/react-dom/components/option.md
Comment (mhaidarhanif) [reply src/content/reference/react-dom/components/option.md:12]: Rule of thumb:

- If it's for the code: don't translate
- If it's not for the code or just a content: can be translated

**Comment by aroyan** [comment]
> @aroyan Outside of @mhaidarhanif's suggestion, all good. Please make the necessary changes.

@mhaidarhanif @resir014 updated to original values

**Review by resir014 — APPROVED** [review]
All good. @mhaidarhanif please review again.

File: src/content/reference/react-dom/components/option.md
Comment (mhaidarhanif) [inline src/content/reference/react-dom/components/option.md:40]:
```suggestion
`<option>` mendukung semua [*props* elemen umum.](/reference/react-dom/components/common#props)
```

File: src/content/reference/react-dom/components/option.md
Comment (mhaidarhanif) [inline src/content/reference/react-dom/components/option.md:42]:
```suggestion
Sebagai tambahan, `<option>` mendukung *props* berikut:
```

Because the original sentence is using "these" and there are some items below this.

**Review by mhaidarhanif — CHANGES_REQUESTED** [review]
Final tiny details to improve

**Review by mhaidarhanif — APPROVED** [review]
Alright, approved 👍

**Pull request merged**
